src/shared/components/EmptyFallback/EmptyFallback.styles.ts

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled'
+
+import { colors, media, sizes } from '@/shared/theme'
+
+import { EmptyFallbackSizes } from './EmptyFallback'
+
+import { Text } from '../Text'
+
+export const Container = styled.div<{ variant?: EmptyFallbackSizes }>`
+  margin: ${({ variant }) => (variant === 'large' ? sizes(20) : sizes(13.5))} auto 0;
+  display: grid;
+  place-items: center;
+
+  ${media.compact} {
+    width: ${({ variant }) => (variant === 'large' ? sizes(90) : 'auto')};
+  }
+
+  > svg {
+    max-width: 650px;
+  }
+`
+
+export const Message = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin-bottom: ${sizes(6)};
```

Comment:
If we apply 40px margin on `Container` I don't think we need this

Reply:
Actually this is the space between message and button, margin on container will not handle this.

Reply:
Let's apply that margin to the button then, `Message` will not always have button after it, right